### ISSUE TITLE
Graphs are missing interpretable labels

### ISSUE THREAD

**Issue by mbsabath** [IQSS/Zelig#293]

**Comment by christophergandrud** [comment]
Hi @mbsabath.  Could you upload an example image and the code producing it?

**Comment by mbsabath** [comment]
yup, it's just a general issue with graph labeling, this example uses the `ls` model.

```R
z.out1 <- zelig(unem ~ gdp + capmob + trade, model = "ls", data = macro)
x <- setx(z.out1)
x <- setx1(x, gdp = 3)
s <- sim(x)
plot(s)
```

Result:
![rplot](https://user-images.githubusercontent.com/10625643/28845735-ff090e44-76d6-11e7-9bf8-5900316a07ae.png)

**Comment by mbsabath** [comment]
X and X1 aren't clear without knowledge of Zelig, the axes aren't labeled, and the scales aren't consistent, making interpretability and comparison difficult

**Comment by mbsabath** [comment]
Submitted Pull Request for this issue [https://github.com/IQSS/Zelig/pull/297](Pull Request 297)
